packages/integrations/vercel/src/routes/callback.ts

```diff
@@ -0,0 +1,54 @@
+import crypto from "crypto";
+import { NextResponse } from "next/server";
+import type { NextRequest } from "next/server";
+
+import { getToken } from "../libs/client";
+import { encrypt } from "../libs/crypto";
+
+export async function GET(req: NextRequest) {
+  const { searchParams } = new URL(req.url);
+  const code = searchParams.get("code");
+  const next = searchParams.get("next");
+  const teamId = searchParams.get("teamId");
+  console.log({ code, next, teamId });
+  if (!code || !next) {
+    throw new Error("Missing `code` or `next` param");
+  }
+
+  /** Get access token from Vercel */
+  const token = await getToken(code);
+
+  // TODO: automatically install log drains to the allowed projects
+
+  const iv = crypto.randomBytes(16);
+  const encryptedToken = encrypt(iv, Buffer.from(token));
+
+  // redirect to vercel's integration page after installation
+  const res = NextResponse.redirect(
+    `https://wwww.openstatus.dev/app/integrations/vercel/configure`,
```

Comment:
Why not using the `next` URL to redirect the user to Vercel's integration page?

Reply:
Because we need to link a Vercel project with a workspace 😭